index.html

```diff
@@ -0,0 +1,207 @@
+            <img src="images/design.svg" class="specifications__table--img" />
+          </div>
+          <h1 class="table__title bold">SLICK AND BEAUTIFUL DESIGN</h1>
+          <h2 class="table__text light">
+            Trendy and fresh design, fits any website.
+          </h2>
+        </section>
+      </div>
+    </section>
+    <!--section collaborate logo brand-->
+    <section class="collaborate__container container">
+      <div class="collaborate__logo block">
+        <img src="images/logos.png" />
+      </div>
+    </section>
+    <!--section article-->
+    <section class="article">
+      <article class="article__one">
+        <img src="images/imac.png" class="article__one--img" />
+      </article>
+      <article class="article__second">
+        <h1 class="article__second_title light">SIMPLE AND BEAUTIFUL</h1>
+        <h2 class="article__second_text">
+          Use Nova theme for your next web project. It is completely
+          customizable so you can Change any of the sections to fit your needs.
+          Nova Theme is Free for kind of use, personal and commercial. Have fun
+          and good luck!
+        </h2>
+      </article>
+    </section>
+    <!--section price-->
+    <section id="pricing" class="price__container container">
+      <div class="price__main_bar block">
+        <h1 class="main__bar_title bold">CHOOSE YOUR PRICING PLAN</h1>
+        <h2 class="main__bar_text light">
+          Pick any of our super affordable pricing plans
+        </h2>
+      </div>
+      <div class="price__list block">
+        <div class="list__first">
+          <h1 class="list__title light">BASIC</h1>
+          <h2 class="price bold">$15</h2>
+          <h3 class="month light">per month</h3>
+          <h4 class="list__offer bold">
+            Up to 7 Projects 2 Additional Developers
+          </h4>
+          <button class="price__button light">Get Started</button>
+        </div>
+        <div class="list__second">
+          <h1 class="list__title light">BASIC</h1>
+          <h2 class="price bold">$15</h2>
+          <h3 class="month light">per month</h3>
+          <h4 class="list__offer bold">
+            Up to 7 Projects 2 Additional Developers
+          </h4>
+          <button class="price__button light">Get Started</button>
+        </div>
+        <div class="list__third">
+          <h1 class="list__title light">BASIC</h1>
+          <h2 class="price bold">$15</h2>
+          <h3 class="month light">per month</h3>
+          <h4 class="list__offer bold">
+            Up to 7 Projects 2 Additional Developers
+          </h4>
+          <button class="price__button light">Get Started</button>
+        </div>
+      </div>
+    </section>
+    <!--section team-->
+    <section id="team" class="team__container container">
+      <div class="team__bar block">THE TEAM</div>
+      <div class="team__catalog block">
+        <div class="catolog__first team-member">
```

Comment:
Zrobiłbym tutaj `ul > li`, gdzie `li` to jeden członek zespołu - dlatego bo mamy listę członków zespołu